src/lib/registration/widgets/registration_code.rb

```diff
@@ -61,20 +61,19 @@ def store
       def register
         if skip?
           log.info("Empty value, skipping registration")
-          return true
+          return false
```

Comment:
why such change? we do not care about return value, so at least document what you want to archieve with it